include/freerdp/server/cliprdr.h

```diff
@@ -68,6 +68,12 @@ struct _cliprdr_server_context
 	void* handle;
 	void* custom;
 
+	/* clipboard capabilities - set by server */
```

Comment:
@mfleisz what do you think about updating the comment to reflect this fact? While reviewing the PR the comment confused me a little ;).